types/api/xverseInscribe/inscription.ts

```diff
@@ -35,6 +40,7 @@ export type InscriptionCreateOrderResponse = {
   commitValue: number;
   commitValueBreakdown: {
     inscriptionValue: number;
+    totalInscriptionValue: number;
```

Comment:
@victorkirov does this get passed to the hook return values?

Reply:
It was not. Good catch 👍  😄